patches/patch1.diff

```diff
@@ -0,0 +1,3160 @@
+diff --git a/data/meson.build b/data/meson.build
```

Comment:
Please add a comment section to the top of the diff saying what this patch is for.

Reply:
Done.